src/mongo_client.ts

```diff
@@ -556,52 +549,34 @@ export class MongoClient extends TypedEventEmitter<MongoClientEvents> {
    * NOTE: presently the operation MUST return a Promise (either explicit or implicitly as an async function)
    *
    * @param options - Optional settings for the command
-   * @param callback - An callback to execute with an implicitly created session
+   * @param withSessionCallback - An callback to execute with an implicitly created session
```

Comment:
The name of the parameter is still callback.  Let's either rename the parameter or revert this change

Reply:
Good catch I say we leave it as is